CONTRIBUTING.md

```diff
@@ -0,0 +1,163 @@
+
+### Tests
+We provide end-to-end tests. You can find them in `fountain/test/test.js`.  
+This contains a series of test executed on every combinations of generated project. You can also choose the stack you want to run the tests on by using `fountain/test/interactive.js`. 
+
+```sh
+$ mocha test/test 
+```
+
+or 
+
+```sh
+$ mocha test/interactive 
+```
+
+The following tests are executed on the generated project: 
+
+- lint the project (both source code and configuration code) regarding its linting rules
+- run project unit tests
+- run `gulp serve` and
+	- test if the list of techs is displayed
+	- test live reload for JS files
+	- test live reload for css/scss/less files
+- run `gulp serve:dist` and test if the list of techs is displayed
+
+Tests are ran by Mocha and we use [Saucelabs](http://saucelabs.com/) and [WebdriverIO](http://webdriver.io/) for remote browser tests.
+
+## <a name="coding"></a> Coding rules
+
+### Ecmascript
+
+In every submodule of FountainJS you can use ES2015 syntax. You can find more about it here https://babeljs.io/docs/learn-es2015/.  
+However, **the code has to be compatible with Node 4 minimum**. If you are using Node 5, we strongly encourage you using `nvm` and test your code on Node 4 as well. You can find which ES2015 features you can use or not here http://node.green/
+
+### Linting rules
+
+We use ESLint as style linter with a shared configuration using [XO](https://github.com/sindresorhus/xo) rules. Please follow this syntax otherwise build tests won't pass. 
+XO configuration is set in `package.json` as an extension of eslint. The generated projects may use additional plugins such as `eslint-config-xo-react`. We use the `space` version of XO which uses 2-space indent.
+
+### Coding style
+
+You are encouraged to follow the coding style of the whole project to keep consistency.
+
+
+## <a name="pr"></a> Pull Requests
+
+If you want to propose a fix or a feature, do as follow: 
+
+- Fork the generator
+- Open a pull request describing the fix/feature
+- Reference the issue when appropriate
+- Make sure your branch can be automatically merged, otherwise rebase your branch
+- Make sure Travis tests are green
+
+### Pull Request template
+
+In any case, every pull request must follow this template: 
+
+>* **Please check if the PR fulfills these requirements**  
+    [ ] Travis tests for the changes are green for the meta-repo  
+    [ ] Travis tests for the changes are green for the repository you contribute to
+
+>* **What kind of change does this PR introduce ?** (Bug fix, feature, docs update, ...)
+
+
+
+>* **What is the current behavior ?** (You can also link to an open issue here)
+
+
+
+>* **What is the new behavior (if this is a feature change) ?**
+
+
+
+>* **Does this PR introduce a breaking change ?** (What changes might users need to make in their application due to this PR?)
+
+
+
+>* **Other information**:
+
+
+## <a name="build"></a> Build tests
```

Comment:
I think github will auto add anchor #build-tests

But maybe this doc will appear on fountainjs.io